src/zope/schema/interfaces.py

```diff
@@ -31,6 +31,10 @@
 from zope.schema._bootstrapfields import Real
 from zope.schema._bootstrapfields import Integral
 from zope.schema._bootstrapfields import Int
+from zope.schema._bootstrapfields import Object
+
+# Import from _bootstrapinterfaces only because other packages will expect
+# to find these interfaces here.
```

Comment:
Why move the comment?

Reply:
Because that comment should only apply to the bootstrap interfaces, not the fields that we use in the implementation of our interfaces.

Reply:
OK